scripts/check-local-config.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var fs = require('fs');
+var path = require('path');
+
+var RELATIVE_FILE = path.join('server', 'config', 'local.json');
+
```

Comment:
Nice! this looks like it ready to go! 

Squash the 2 commits into one and remove this extra line break while you are at it.

R+.